src/Kinto.elm

```diff
@@ -431,34 +404,98 @@ errorDecoder =
         (Decode.field "error" Decode.string)
 
 
-{-| Change the error from an `Http.Error` to an `Error`.
--}
-toResponse : Result Http.Error a -> Result Error a
-toResponse response =
-    response
-        |> Result.mapError extractError
-
-
 {-| Extract an `Error` from an `Http.Error`.
 -}
-extractError : Http.Error -> Error
```

Comment:
I believe you haven't removed the associated documentation (the two lines just above)